utils.py

```diff
@@ -990,13 +1092,17 @@ def compute_list_difference(list_a, list_b):
     return list(set(list_a) - set(list_b))
 
 
-class OrderedCounter(collections.Counter, collections.OrderedDict):
+# Ignoring type-arg because error thrown is 'Missing type parameters for generic
+# type "OrderedDict"' but here we don't need to specify this.
+class OrderedCounter(
+        collections.Counter, collections.OrderedDict): # type: ignore[type-arg]
     """Counter that remembers the order elements are first encountered."""
 
     pass
 
 
 def grouper(iterable, chunk_len, fillvalue=None):
+    # type: (Iterable[T], int, Any) -> Iterable[Iterable[T]]
```

Comment:
I think we can force the `fillvalue` to be the same type as the stuff in iterable.
```suggestion
    # type: (Iterable[T], int, Optional[T]) -> Iterable[Iterable[T]]
```

Reply:
Done.